### ISSUE TITLE
Support for Firefox Mobile

### ISSUE THREAD

**Issue by kyuucr** [mrdokenny/History-AutoDelete#19]
As I checked on the AMO, this extension can't be installed on Firefox Mobile. Is it possible to enable support for Firefox Mobile?

**Comment by mrdokenny** [comment]
Yes it will be possible, once Firefox implements the APIs that I need for mobile:
BrowserAction
History
Tabs

[https://trello.com/b/PC9kB14s/webextensions-roadmap](WebExtension Roadmap)

**Comment by kyuucr** [comment]
FYI, updates on the requirements:

- BrowserAction seems to have been implemented partly on mobile [source](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/browserAction#Browser_compatibility)
- History has not been implemented yet
- Options UI Support has been [resolved](https://bugzilla.mozilla.org/show_bug.cgi?id=1302504)
